tests/bruno/e2e/open-use-close-a-channel/02-open-channel.bru

```diff
@@ -23,7 +23,8 @@ body:json {
     "params": [
       {
         "peer_id": "{{NODE1_PEERID}}",
-        "funding_amount": "0xba43b7400"
+        "funding_amount": "0xba43b7400",
+        "commitment_delay_epoch": "0x30001000002"
```

Comment:
This seems to be 2 and 1/3 epochs. DId you mean to use `0x20001000003` (if we are using big endian to de-serialize u64)?

Reply:
3.5 epochs is presented as 3(E) and 1(I)/2(L): (L << 40) | (I << 24) | E = 0x20001000003,

ref: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0017-tx-valid-since/e-i-l-encoding.png

Reply:
Yeah. But the number in this bruno file is `0x30001000002`, instead of the number in your reply, `0x20001000003`.

Reply:
sorry, I thought that you are ref to https://github.com/nervosnetwork/fiber/blob/568eec50e27f8e4cfda3b1d2b9cb48fb810b93f3/tests/bruno/e2e/open-use-close-a-channel/README.md?plain=1#L30 , fixed in new commit.